Stratis.Bitcoin.Features.LightWallet/LightWalletFeePolicy.cs

```diff
@@ -15,10 +15,11 @@ namespace Stratis.Bitcoin.Features.LightWallet
     public class LightWalletFeePolicy : IWalletFeePolicy
     {
         private readonly Money maxTxFee;
-        private readonly HttpClient httpClient;
+        private static readonly HttpClient HttpClient = new HttpClient();
```

Comment:
We try as much as possible to avoid initializing fields outside of the constructor.
It's a coding preference, but it prevents the debugger to jump up and down during debugging.
Look at item 17 of this doc: https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/Documentation/coding-style.md